### PR TITLE
[SYNCOPE-1597] Parametrized console to enable non role-based customization

### PR DESCRIPTION
**Pull request by andrea-patricelli**

### CODE REVIEW

**Review by ilgrosso — CHANGES_REQUESTED** [review]
The following file is not updated:

```
docker/console/src/main/resources/console.properties.template
```

File: client/console/src/main/resources/console.properties [hunk 1]
```diff
@@ -56,3 +56,5 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
+default.any.panel.class=org.apache.syncope.client.console.panels.AnyPanel
```
Comment (ilgrosso) [inline client/console/src/main/resources/console.properties:60]: Please move this parameter upwards, right after the section with pages.

File: fit/console-reference/src/main/resources/console.properties [hunk 2]
```diff
@@ -56,3 +56,6 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
+default.any.panel.class=org.apache.syncope.client.console.panels.AnyPanel
+
```
Comment (ilgrosso) [inline fit/console-reference/src/main/resources/console.properties:61]: Please move this parameter upwards, right after the section with pages.

File: fit/core-reference/src/test/resources/console.properties [hunk 3]
```diff
@@ -56,3 +56,6 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
+default.any.panel.class=org.apache.syncope.client.console.panels.AnyPanel
```
Comment (ilgrosso) [inline fit/core-reference/src/test/resources/console.properties:60]: Please move this parameter upwards, right after the section with pages.

**Review by ilgrosso — CHANGES_REQUESTED** [review]

File: client/console/src/main/resources/console.properties [hunk 4]
```diff
@@ -56,3 +58,4 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
```
Comment (ilgrosso) [inline client/console/src/main/resources/console.properties:61]: Remove this blank line

File: docker/console/src/main/resources/console.properties.template [hunk 5]
```diff
@@ -56,3 +58,4 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
```
Comment (ilgrosso) [inline docker/console/src/main/resources/console.properties.template:61]: Remove this blank link

File: fit/console-reference/src/main/resources/console.properties [hunk 6]
```diff
@@ -56,3 +58,5 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
```
Comment (ilgrosso) [inline fit/console-reference/src/main/resources/console.properties:61]: Remove these blank lines

File: fit/core-reference/src/test/resources/console.properties [hunk 7]
```diff
@@ -56,3 +58,5 @@ security.headers.Strict-Transport-Security=max-age=31536000; includeSubDomains;
 security.headers.X-Content-Type-Options=nosniff
 security.headers.X-Frame-Options=sameorigin
 #security.headers.Content-Security-Policy=default-src https:
+
```
Comment (ilgrosso) [inline fit/core-reference/src/test/resources/console.properties:61]: Remove these blank linkes

**Review by ilgrosso — CHANGES_REQUESTED** [review]

**Comment by ilgrosso** [comment]
@andrea-patricelli as said above, `docker/console/src/main/resources/console.properties.template` is missing

**Comment by andrea-patricelli** [comment]
> @andrea-patricelli as said above, `docker/console/src/main/resources/console.properties.template` is missing

Added with last commit. Going to remove also blank lines

**Review by ilgrosso — APPROVED** [review]

**Pull request merged**